content/400-reference/200-api-reference/200-command-reference.mdx

```diff
@@ -1285,3 +1285,39 @@ prisma studio --browser firefox
 ```terminal
 prisma studio --browser none
 ```
+
+### <inlinecode>package.json</inlinecode> entry options
+
+#### <inlinecode>schema</inlinecode> path
+
+The path to the desired `schema.prisma` file can be specified with the `prisma.schema` entry in the `package.json` file. The path defines the file the Prisma CLI should use when you run any of the CLI commands. Both absolute and relative paths are supported.
+
+```json file="package.json"
+{
+  "name": "my-project",
+  "version": "1.0.0",
+  "prisma": {
+    "schema": "./custom-path-to-schema/schema.prisma"
+  }
+}
+```
+
+This is available from version 2.7.0 and later.
+
+#### <inlinecode>seed</inlinecode> option
```

Comment:
```suggestion
#### <inlinecode>seed</inlinecode> command
```

Reply:
I think this refers to `prisma.seed` "option" (?) in `package.json`, not the `db seed` command?

Reply:
I've removed the schema and option keywords.